src/locales/pl-pl.ts

```diff
@@ -59,6 +64,9 @@ const plPL: Locale = {
   heading: {
     backLinkAriaLabel: () => "Wstecz",
   },
+  hintString: {
+    primaryHintString: () => "Wejście zawiera licznik znaków",
```

Comment:
```suggestion
    primaryHintString: () => "Pole zawiera licznik znaków",
```

Reply:
I think the `hintString` should be contained inside the `characterCount` field.